packages/features/form-builder/testUtils.ts

```diff
@@ -0,0 +1,234 @@
+    },
+    saveField: ({ dialog }: { dialog: ReturnType<typeof within> }) => {
+      fireEvent.click(dialog.getByTestId("field-add-save"));
+    },
+    openFieldTypeDropdown: ({ dialog }: { dialog: TestingLibraryElement }) => {
+      fireEvent.keyDown(dialog.getByTestId("test-field-type"), { key: "ArrowDown", code: "ArrowDown" });
+    },
+    fieldTypeDropdown: {
+      queryOptionForFieldType: ({
+        dialog,
+        fieldType,
+      }: {
+        dialog: TestingLibraryElement;
+        fieldType: string;
+      }) => {
+        const fieldTypeEl = within(dialog.getByTestId("test-field-type"));
+        return fieldTypeEl.queryByTestId(`select-option-${fieldType}`);
+      },
+      queryAllOptions: ({ dialog }: { dialog: TestingLibraryElement }) => {
+        const fieldTypeElementWithOptions = within(dialog.getByTestId("test-field-type").parentElement);
+        return fieldTypeElementWithOptions.queryAllByTestId(/^select-option-/);
+      },
+    },
+    selectFieldType: ({ dialog, fieldType }: { dialog: TestingLibraryElement; fieldType: string }) => {
+      pageObject.dialog.openFieldTypeDropdown({ dialog });
+      fireEvent.click(dialog.getByTestId(`select-option-${fieldType}`));
+    },
+    fillInFieldIdentifier: ({
+      dialog,
+      identifier,
+    }: {
+      dialog: TestingLibraryElement;
+      identifier: string;
+    }) => {
+      fireEvent.change(dialog.getAllByRole("textbox")[0], { target: { value: identifier } });
+    },
+    fillInFieldLabel: ({ dialog, label }: { dialog: TestingLibraryElement; label: string }) => {
+      fireEvent.change(dialog.getAllByRole("textbox")[1], { target: { value: label } });
+    },
+    close: ({ dialog }: { dialog: TestingLibraryElement }) => {
+      fireEvent.click(dialog.getByTestId("dialog-rejection"));
+    },
+  },
+  queryDeleteButton: ({ identifier }: { identifier: string }) => {
+    const field = getFieldInTheList({ identifier });
+    return field.queryByTestId("delete-field-action");
+  },
+  getDeleteButton: ({ identifier }: { identifier: string }) => {
+    const field = getFieldInTheList({ identifier });
+    return field.queryByTestId("delete-field-action");
+  },
+  deleteField: ({ identifier }: { identifier: string }) => {
+    const field = getFieldInTheList({ identifier });
+    fireEvent.click(field.getByTestId("delete-field-action"));
+  },
+  queryToggleButton: ({ identifier }: { identifier: string }) => {
+    const field = getFieldInTheList({ identifier });
+    return field.queryByTestId("toggle-field");
+  },
+  getToggleButton: ({ identifier }: { identifier: string }) => {
+    const field = getFieldInTheList({ identifier });
+    return field.getByTestId("toggle-field");
+  },
+  toggleField: ({ identifier }: { identifier: string }) => {
+    const field = getFieldInTheList({ identifier });
+    fireEvent.click(field.getByTestId("toggle-field"));
+  },
+};
+
+export const verifier = {
+  verifyFieldAddition: async (props: FieldProps) => {
```

Comment:
It holds the same stuff that was there in now deleted utils.ts. But things have been renamed and pageObject is being used to make the code readable.